lib/playground.dart

```diff
@@ -75,6 +75,8 @@ class Playground extends EditorUi implements GistContainer, GistController {
       DElement(querySelector('#web-output-label')!);
   final MDCSwitch _nullSafetySwitch =
       MDCSwitch(querySelector('#null-safety-switch'));
+  final Element? _channelSwitch = querySelector('#channel-switch');
+  late final Future<MDCMenu> _channelsMenu = _initChannelsMenu();
```

Comment:
Can we make the channel fetch part of the initialization instead? This initialization logic is a a little difficult to work with. (I'm trying to add checkboxes to the menu)